fe/src/pages/About/About.tsx

```diff
@@ -1,9 +1,11 @@
+import { Flex, Text } from '@chakra-ui/react';
+
 function About(): JSX.Element {
   // @TODO - Add some actual content here.
   return (
-    <div>
-      <h1>About Page</h1>
-    </div>
+    <Flex w="100%" minH="100vh" justifyContent="center" flexDirection="column" alignItems="center">
```

Comment:
This seems like a hacky fix for the sidebar. Sidebar should not depend on page to be a flex element. If any reviewers have other ideas on this let us know. 🤔

Reply:
cc: @Orzelius @Kasparsu

Reply:
it doesnt need flex. I just but it there to center stuff